library/std/src/lib.rs

```diff
@@ -282,6 +282,7 @@
 #![feature(maybe_uninit_ref)]
 #![feature(maybe_uninit_slice)]
 #![feature(min_specialization)]
+#![feature(mutex_arc)]
```

Comment:
Is this only used for tests? If so, maybe:
```suggestion
#![cfg_attr(feature(mutex_arc))]
```
?